### ISSUE TITLE
Coxeter groups

### ISSUE THREAD

**Issue by jwj61** [LMFDB/lmfdb#6195]
From the feedback page:

I love this project, thank you for all the hard work!!!!!

It would be extremely useful to a lot of researchers if the abstract groups part of the website contained the finite Coxeter reflection groups.  I managed to locate some of them, perhaps these could be added to the `interesting groups' page for others to find?  It would also be good to add all of the missing ones (at least as far as order bounds make sense)

I haven't checked for An, Bn, Dn for n<4 (but it would be good to include B3 and D3)

H3: https://beta.lmfdb.org/Groups/Abstract/120.35

A4: https://beta.lmfdb.org/Groups/Abstract/120.34
B4: https://beta.lmfdb.org/Groups/Abstract/384.5602
D4: https://beta.lmfdb.org/Groups/Abstract/192.1493
F4: https://beta.lmfdb.org/Groups/Abstract/1152.157478
H4: https://beta.lmfdb.org/Groups/Abstract/14400.a

A5: Is Sym(6)  (In general An=Sym(n+1) so this is less important)
B5: https://beta.lmfdb.org/Groups/Abstract/3840.ch
D5: https://beta.lmfdb.org/Groups/Abstract/1920.240996

A6 is Sym(7)
I could not find B6=C2wrS6 or D6 (index two in B6) in the database (in general Bn=C2wrSn and Dn is index 2
E6: https://beta.lmfdb.org/Groups/Abstract/51840.b

I could not find E7 but it is Sp(6,2)*C2 and Sp(6,2) can be found here:
https://beta.lmfdb.org/Groups/Abstract/1451520.a 

I could not locate E8 (it is a group of shape 2.GO+(8,2) but can easily be constructed with a Coxeter presentation).

I hope you find the time to add this to the database.

**Comment by roed314** [comment]
I agree, and I'd also like to add the Coxeter presentation in the "Constructions" section when appropriate.
